src/freenet/node/FailureTable.java

```diff
@@ -287,8 +287,8 @@ public boolean isExpired() {
 	 */
 	public void onFound(KeyBlock block) {
 		if(logMINOR) Logger.minor(this, "Found "+block.getKey());
-		if(!(node.enableULPRDataPropagation || node.enablePerNodeFailureTables)) {
-			if(logMINOR) Logger.minor(this, "Ignoring onFound because enable ULPR = "+node.enableULPRDataPropagation+" and enable failure tables = "+node.enablePerNodeFailureTables);
+		if(!(node.enableULPRDataPropagation() || node.enablePerNodeFailureTables())) {
```

Comment:
Probably there should be node.isULPRDataPropagationEnabled()

Reply:
That method name would also be fine for me. Shall I rename it and add a corresponding commit?